### DISCUSSION TITLE
Collection Relationship Property Not Reserializing

### DISCUSSION THREAD

**Discussion by maweitzman** [livewire/livewire#4439] [1. Help]
So we are reworking our front end architecture to not use livewire in children components, rather just using it in parent components who pass their properties down to the children.

Since we started doing this, some of the children components are having issues. When the parent iterates through a collection and renders children, any properties passed down that have relationships are not preserved and throw errors. Is there a way to solve this or rework it?

@joshhanley I don't know if this is a simple fix?

**Comment by joshhanley** [comment]
@maweitzman I'm not exactly sure what you mean by not using Livewire in child components. Are you talking about not using Livewire child components or not using Livewire in blade components?

Could you share your code which reproduces the issue with as little code as possible so we can have a better idea of what you are trying to do? And explain in more detail what is happening exactly?

Thanks!
